azurerm/resource_arm_application_gateway.go

```diff
@@ -2620,6 +2660,7 @@ func expandApplicationGatewayWafConfig(d *schema.ResourceData) *network.Applicat
 		FileUploadLimitInMb:    utils.Int32(int32(fileUploadLimitInMb)),
 		RequestBodyCheck:       utils.Bool(requestBodyCheck),
 		MaxRequestBodySizeInKb: utils.Int32(int32(maxRequestBodySizeInKb)),
+		Exclusions:             &exclusions,
```

Comment:
We want to have the above code in a separate expand function, similar to what I suggested in the other review